lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -498,8 +498,10 @@ newtype PostExternalTransactionData = PostExternalTransactionData
     { payload :: ByteString
     } deriving (Eq, Generic, Show)
 
-newtype ApiFee = ApiFee
-    { amount :: (Quantity "lovelace" Natural)
+data ApiFee = ApiFee
+    { amount :: !(Quantity "lovelace" Natural)
+    , estimatedMin :: !(Quantity "lovelace" Natural)
+    , estimatedMax :: !(Quantity "lovelace" Natural)
```

Comment:
I guess the motivation for keeping `amount` around is backwards compatibility?

Perhaps an alternative option is to leave the existing API call(s) alone, and introduce a completely new API call that uses a new datatype:

```hs
data ApiFeeEstimate = ApiFeeEstimate
    { estimateMin :: !(Quantity "lovelace" Natural)
    , estimateMax :: !(Quantity "lovelace" Natural)
    } deriving (Eq, Generic, Show)
```

Reply:
Marking as resolved, as we decided to break backwards compatibility.

Reply:
> Perhaps an alternative option is to leave the existing API call(s) alone (adding a deprecated tag), and introduce a completely new API call that uses a new datatype:

Marking resolved, as we decided to break backwards compatibility.